pushy/src/main/java/com/relayrides/pushy/apns/ApnsClientBuilder.java

```diff
@@ -349,6 +354,34 @@ public ApnsClientBuilder setGracefulShutdownTimeout(final long gracefulShutdownT
     }
 
     /**
+     * Sets the buffer allocator for the underlying netty channel
+     *
+     * @param byteBufferAllocator the buffer allocator to use for the underlying netty channel
+     *
+     * @return a reference to this builder
+     *
+     * @since 0.9
+     */
+    public ApnsClientBuilder setByteBufferAllocator(ByteBufAllocator byteBufferAllocator) {
+        this.byteBufferAllocator = byteBufferAllocator;
+        return this;
+    }
+
+    /**
+     * Sets the buffer usage watermark range for the underlying netty channel
```

Comment:
Can we be a little more descriptive here, too? This seems like a good place to mention the existence of `ClientBusyException`, for example.